res/header.asm

```diff
@@ -118,7 +151,15 @@
 ; Desc  - If stack[-2] < stack[-1], push isTrue on stack, else isFalse
 ; -----------------------------------------------------------------------------
 %macro O_LSS 0
-  ; TBD
+  POP	RBX					; Get 'b' from stack
+  POP	RAX					; Get 'a' from stack
+  CMP	RAX, RBX			; a ?? b
+  JNB	%%a_geq_b
```

Comment:
`JNB` is for unsigned data, we need to compare signed values as well, so we need `JG/JNLE`
Source: https://www.tutorialspoint.com/assembly_programming/assembly_conditions.htm